lib/common/models/BlockchainLockModel.ts

```diff
@@ -9,8 +9,8 @@ export default interface BlockchainLockModel {
   amountLocked: number;
 
   /** At this transaction time the lock is no longer valid */
-  lockEndTransactionTime: number;
+  unlockTransactionTime: number;
 
-  /** The wallet where the amount goes upon unlock */
-  linkedWalletAddress: string;
+  /** The destination where the amount goes upon unlock */

```

Comment:
Consider to describe it generally without referring to implementation detail.
```suggestion
  /** The owner of the lock. */

```